ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v3/fileinput/clientlibs/site/js/fileinputwidget.js

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+/**
+ * This class is responsible for interacting with the file input widget. It implements the file preview,
+ * file list, handling invalid file size, file name, file mime type functionality
+ */
+
+if (typeof window.FileInputWidget === 'undefined') {
+    window.FileInputWidget = class extends FormView.FormFileInputWidget {
+
+        constructor(params) {
+            super(params);
+        }
+
+        attachEventHandlers(widget, dragArea, model) {
```

Comment:
you should call the super.attachEvenHandlers() and only add those code here which is specific to v3

Reply:
Implemented!